libgalois/include/galois/graphs/FileGraph.h

```diff
@@ -98,6 +98,10 @@ class FileGraph {
   //! adjustments to edge index when we load only part of a graph
   uint64_t edgeOffset;
 
+  //! If initialized, this array stores node degrees in memory for fast access
+  //! via a getDegree function
+  LargeArray<uint64_t> node_degrees;
```

Comment:
Instead of node degrees, I would prefer if it is the prefix sum array (indices) of the CSR directly so that it can be used for loading edges subsequently.